test/interactions/panZoomInteractionTests.ts

```diff
@@ -1,39 +1,119 @@
-      xScale.domain([0, SVG_WIDTH / 2]).range([0, SVG_WIDTH]);
-      yScale = new Plottable.Scales.Linear();
-      yScale.domain([0, SVG_HEIGHT / 2]).range([0, SVG_HEIGHT]);
-      panZoomInteraction = new Plottable.Interactions.PanZoom();
-      panZoomInteraction.addXScale(xScale);
-      panZoomInteraction.addYScale(yScale);
-      panZoomInteraction.attachTo(component);
-
-      eventTarget = component.background();
+  describe("PanZoom Interaction", () => {
+
+    describe("Scale setting", () => {
+      let xScale: Plottable.QuantitativeScale<number>;
+      let yScale: Plottable.QuantitativeScale<number>;
+      let panZoomInteraction: Plottable.Interactions.PanZoom;
+
+      beforeEach(() => {
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+
+        panZoomInteraction = new Plottable.Interactions.PanZoom();
+      });
+
+      it("can set scales", () => {
+        assert.deepEqual(panZoomInteraction.xScales(), [], "no xScales exist by default");
+        assert.deepEqual(panZoomInteraction.yScales(), [], "no yScales exist by default");
+
+        assert.strictEqual(panZoomInteraction.addXScale(xScale), panZoomInteraction, "setting the xScale returns the interaction");
+        assert.strictEqual(panZoomInteraction.addYScale(yScale), panZoomInteraction, "setting the yScale returns the interaction");
+
+        assert.deepEqual(panZoomInteraction.xScales(), [xScale], "xScale has been added");
+        assert.deepEqual(panZoomInteraction.yScales(), [yScale], "yScale has been added");
+      });
+
+      it("can remove scales", () => {
+        panZoomInteraction.addXScale(xScale);
+        panZoomInteraction.addYScale(yScale);
+
+        assert.deepEqual(panZoomInteraction.xScales(), [xScale], "xScale has been added");
+        assert.deepEqual(panZoomInteraction.yScales(), [yScale], "yScale has been added");
+
+        assert.strictEqual(panZoomInteraction.removeXScale(xScale), panZoomInteraction, "removing the xScale returns the interaction");
+        assert.strictEqual(panZoomInteraction.removeYScale(yScale), panZoomInteraction, "removing the yScale returns the interaction");
+
+        assert.deepEqual(panZoomInteraction.xScales(), [], "xScale has been removed");
+        assert.deepEqual(panZoomInteraction.yScales(), [], "yScale has been removed");
+
+      });
+
+      it("can add set the xScales in batch", () => {
+        let xScale2 = new Plottable.Scales.Linear();
+
+        panZoomInteraction.addXScale(xScale);
+        panZoomInteraction.addXScale(xScale2);
+        let xScales = panZoomInteraction.xScales();
+
+        panZoomInteraction.xScales([]);
+        assert.deepEqual(panZoomInteraction.xScales(), [], "scales can be removed with the xScales call");
+
+        panZoomInteraction.xScales([xScale, xScale2]);
+        assert.deepEqual(xScales, panZoomInteraction.xScales(), "setting and adding x scales result in the same behavior");
```

Comment:
this is kinda weird 
I would prefer you test `panZoomInteraction.xScales()` returns `[xScale, xScale2]` after calling `addXScale` twice as well as when calling `xScales([xScale, xScale2])` instead of comparing the return value against each other
also, `panZoomInteraction.xScales()` should be the first argument of assert